netbox/virtualization/choices.py

```diff
@@ -5,6 +5,23 @@
 # VirtualMachines
 #
 
+class MemoryUnitChoices(ChoiceSet):
```

Comment:
`ChoiceSet` should always declare a `CHOICES` constant. Rather than trying to combine them, we should create a separate `ChoiceSet` class for each purpose. The `UNIT_*` constants can be defined outside of the classes for reuse to cut down on boilerplate.

Reply:
Done.